libraries/location.rb

```diff
@@ -21,6 +21,8 @@ def name
 
   def id
     current_value['id']
+  rescue
```

Comment:
You should only rescue the specific error class here - you don't want to do this for all exceptions.